docs/deploy-etcd.md

```diff
@@ -9,11 +9,15 @@
 kube-system:tiller`.
 - Run `helm init --service-account tiller` to initialize the helm component.
 
-### Deploy etcd operator and create an etcd cluster
+### Deploy etcd operator
 
 - Make sure the `compose` namespace exists on your cluster.
 - Run `helm install --name etcd-operator stable/etcd-operator --namespace compose` to install the etcd-operator chart.
-- Create an etcd cluster definition like this one in a file named compose-etcd.yaml:
+
+### Option1: Create an ETCD cluster (for quick evaluation)
+
+This will create an ETCD cluster quickly, but without High Availability, nor persistent storage, and that can be accessed without authentication.
```

Comment:
From my understanding re #71 this also implies you cannot restart anything, is that right? If so then worth spelling that out explicitly I think.